tests/version.cpp

```diff
@@ -28,8 +28,7 @@ namespace cps::version::test {
             case Operator::lt:
                 return "<";
             default:
-                CPS_UNREACHABLE("How did you get here?");
-                return ""; // Suppress missing return statement warning
+                abort();
```

Comment:
```suggestion
                std::abort();
```
Maybe use `std::abort` in both places just to be consistent?